### PR TITLE
Capif aef demo

### PR DESCRIPTION
**Pull request by jorgems101**
After Demo, many improvements are included in this branch. We must merge this to develop and create new release 2.1.

### CODE REVIEW

File: tests/resources/common/basicRequests.robot [hunk 1]
```diff
@@ -153,37 +156,46 @@ Register User At Jwt Auth
...
 Clean Test Information By HTTP Requests
-    Create Session    jwtsession    ${CAPIF_HTTP_URL}   verify=True
+    Create Session    jwtsession    ${CAPIF_HTTP_URL}    verify=True
 
     ${resp}=    DELETE On Session    jwtsession    /testdata
     Should Be Equal As Strings    ${resp.status_code}    200
 
-    ${resp}=    DELETE On Session    jwtsession    /certdata
-    Should Be Equal As Strings    ${resp.status_code}    200
+    # ${resp}=    DELETE On Session    jwtsession    /certdata
```
Comment (raxonros) [inline tests/resources/common/basicRequests.robot]: Si es código comentado mejor quitarlo no?

File: tests/resources/common/basicRequests.robot [hunk 2]
```diff
@@ -3,10 +3,11 @@ Documentation       This resource file contains the basic requests used by Capif
 
 Library             RequestsLibrary
 Library             Collections
+Library             OperatingSystem
 
 
 *** Variables ***
-# ${NGINX_HOSTNAME}       http://localhost:8080
+# ${NGINX_HOSTNAME}    http://localhost:8080
```
Comment (raxonros) [inline tests/resources/common/basicRequests.robot]: Código comentado mejor quitarlo

File: tests/features/CAPIF Api Publish Service/capif_api_publish_service.robot [hunk 3]
```diff
@@ -18,18 +18,18 @@ Publish API by Authorised API Publisher
...
-    ...    sign-csr
-    ...    json=${request_body}
-    ...    server=${CAPIF_HTTP_URL}
-    ...    verify=ca.crt
-    # ...    access_token=${register_user_info['access_token']}
-    Status Should Be    201    ${resp}
-
-    # Store dummy signede certificate
-    Store In File    ${PUBLISHER_USERNAME}.crt    ${resp.json()['certificate']}
+    # # Sign certificate
```
Comment (raxonros) [inline tests/features/CAPIF Api Publish Service/capif_api_publish_service.robot]: Código comentado mejor quitarlo

File: tests/resources/common/basicRequests.robot (hunk 1 above)
Comment (jorgems101) [reply tests/resources/common/basicRequests.robot]: CIerto, ahora mismo lo quito

File: tests/resources/common/basicRequests.robot (hunk 2 above)
Comment (jorgems101) [reply tests/resources/common/basicRequests.robot]: lo quito

File: tests/features/CAPIF Api Publish Service/capif_api_publish_service.robot (hunk 3 above)
Comment (jorgems101) [reply tests/features/CAPIF Api Publish Service/capif_api_publish_service.robot]: quitado

**Review by raxonros — COMMENTED** [review]
LGTM

File: services/easy_rsa/revoke_demo_users.sh [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+# /root/EasyRSA-3.0.4/easyrsa --batch revoke $DUMMY
+# rm -rf /root/$DUMMY.csr
+# rm -rf /root/pki/issued/$DUMMY.crt
+# rm -rf /root/pki/reqs/$DUMMY.req
+
+# /root/EasyRSA-3.0.4/easyrsa --batch revoke $CUSTOMER_EXPOSER
+# rm -rf /root/$CUSTOMER_EXPOSER.csr
+# rm -rf /root/pki/issued/$CUSTOMER_EXPOSER.crt
+# rm -rf /root/pki/reqs/$CUSTOMER_EXPOSER.req
```
Comment (alemosan1) [inline services/easy_rsa/revoke_demo_users.sh]: I would remove comments if we won't use them

Comment (jorgems101) [reply services/easy_rsa/revoke_demo_users.sh]: Sure. Removing

**Review by alemosan1 — DISMISSED** [review]
LGTM

**Review by raxonros — DISMISSED** [review]

**Review by alemosan1 — APPROVED** [review]

**Pull request merged**